### PR TITLE
Update s1slc docs to include two new params

### PR DESCRIPTION
**Pull request by nekhtiari**
Pull Request
============
### Scope of the PR

This PR adds two extra params to the s1slc params description.

### Checklist

- [ ] Checked spelling and grammar
- [ ] Provided code samples where relevant
- [x] Checked the output of `make html` to ensure new content displays correctly

### CODE REVIEW

**Review by teoEnache — APPROVED** [review]
looks good

File: source/up42-blocks/data/sobloo-s1-slc.rst [hunk 1]
```diff
@@ -40,6 +40,9 @@ For more information, please read the section :ref:`Data source query filters  <
 * ``ids``: An array of image identifiers. The S1 identifiers are described here: https://sentinel.esa.int/web/sentinel/user-guides/sentinel-1-sar/naming-conventions; the file extension is omitted. By defining the ``ids`` filter you specify unambiguously which images to retrieve based solely on the given ID(s). The ``ids`` filter overrides all other filters, e.g., ``intersects``, ``limit`` and/or ``time``.
 * ``acquisition_mode``: A string allowing querying for specific acquisition modes, see https://sentinel.esa.int/web/sentinel/user-guides/sentinel-1-sar/acquisition-modes. for details.
 * ``orbit_direction``: The satellite orbit direction. One of ``null``, ``ASCENDING`` or ``DESCENDING``. Default is ``null``, which queries both ascending and descending.
+* ``mission_code``: The satellite mission code(id). One of ``S1A``, or ``S1B``. Default is ``null``, which queries both mission codes.
+* ``orbit-relative-number``: The satellite orbit relative number.
```
Comment (markusuwe) [inline source/up42-blocks/data/sobloo-s1-slc.rst:44]: Can we add an example value?

**Review by markusuwe — APPROVED** [review]
LGTM

File: source/up42-blocks/data/sobloo-s1-slc.rst (hunk 1 above)
Comment (nekhtiari) [reply source/up42-blocks/data/sobloo-s1-slc.rst]: added!

**Pull request merged**